### PR TITLE
translations/ml: Add Malayalam translations.

### PR DESCRIPTION
**Pull request by NewtronReal**
I made this translation understandable to a native as much as I can. Hope you won't feel it like a literary thing. If you are struggling to understand any of these translations please report it.

Preexisting Malayalam words but new to digital field:

ഇന്‍സ്റ്റാളിടം          install location
കൊണ്ടുവരുക        to import
കടത്താള്‍             store page
പ്രവര്‍ത്തിക്കുന്നു      enabled
പ്രവര്‍ത്തിക്കുന്നില്ല   disabled
എടുത്തു നീക്കുക      to uninstall
ഇറക്കിയെടുക്കുക    to download

for more word meanings visit my github page.

### CODE REVIEW

**Comment by adityaruplaha** [comment]
Great to see Indic languages bring supported as an Indian. Keep up the good work!

**Comment by NewtronReal** [comment]
@adityaruplaha 
I will try to translate those two files soon. Thanks for your appreciation. I am not sure about how much it will support the launcher. Because, I am using a potato which takes a while to compile the source. I never tried to compile it from source. What about the fonts?? Can I suggest default fonts for my language??

**Comment by adityaruplaha** [comment]
I don't think you need to compile from source really, we have tests on every PR. And every single bit helps!

Also the fonts used should support Malayalam automatically, but I'll double check.

File: src/components/UI/LanguageSelector/index.tsx [hunk 1]
```diff
@@ -39,6 +40,7 @@ export default function LanguageSelector({
     es: '🇪🇸',
     fr: '🇫🇷',
     hu: '🇭🇺',
+    ml: '🇮🇳',
```
Comment (adityaruplaha) [inline src/components/UI/LanguageSelector/index.tsx:43]: I'm not sure about putting a flag here. India has wayyyyyyyyyyyyyyy too many languages.

**Review by adityaruplaha — APPROVED** [review]
LGTM overall. Left a minor comment, need not be addressed.

**Comment by adityaruplaha** [comment]
Also you might want to join our Discord to keep up with the latest developments, and we might need help with translations in the future too! :)

**Comment by adityaruplaha** [comment]
I'm sorry, looks like I made a mistake while resolving conflicts. Kindly fix and we can merge.

File: src/components/UI/LanguageSelector/index.tsx (hunk 1 above)
Comment (NewtronReal) [reply src/components/UI/LanguageSelector/index.tsx:43]: ml:"മ🇮🇳";

what about putting first letter of language name in that languages script before the nation flag for denoting the language (only for nations containing more than one official languages)??

**Comment by NewtronReal** [comment]
@adityaruplaha 
sorry for late response. Conflicts now seems to have fixed forget to check my suggestion about the problem of flags.

File: src/components/UI/LanguageSelector/index.tsx (hunk 1 above)
Comment (adityaruplaha) [reply src/components/UI/LanguageSelector/index.tsx]: Yeah no imo it'll be better to just use the Indian flag.

**Review by adityaruplaha — APPROVED** [review]

**Comment by adityaruplaha** [comment]
Everything looks good, merging as soon as checks finish.

**Comment by flavioislima** [comment]
@NewtronReal @adityaruplaha 
Noticed that two JSON files are with issues and were missing during the review.
please open a new PR to fix:
`login.json` **line 16** and `translation.json` **line 9**.

**Comment by adityaruplaha** [comment]
Ah crap how did I miss those, really sorry about it.

**Comment by flavioislima** [comment]
> 
> 
> Ah crap how did I miss those, really sorry about it.

No Worries. I discovered that when I was configuring the project on Weblate and on got the error when importing the existing translations.

**Comment by NewtronReal** [comment]
@flavioislima oops 😬. Really sorry about that I'll check and fix that. Thanks.

**Comment by flavioislima** [comment]
Hi!
@NewtronReal 

I guess this translation is getting outdated, can you fix it using weblate?
https://hosted.weblate.org/languages/ml/heroic-games-launcher/

It might show 98% but is not because its counting the English translations in between.
If you check the components individually it will show that.

Thanks!

**Pull request merged**